config/default/test/test-helpers.js

```diff
@@ -0,0 +1,43 @@
+
+const MAX_DAYS_IN_PREGNANCY = 42 * 7;  // 42 weeks
+
+function getRangeForTask(task) {
+    const a = task.triggers.map(t => {
+        t = task.triggerUnit === "weeks" ? t * 7 : t;
+        const start = t - task.pre;
+        const end = t + task.post;
+        return range(start, end);
+    });
+
+    // return the flattened 1D array, with distinct members
+    return [...new Set([].concat.apply([], a))];
+}
+
+//1 day before start, start day, end day, 1 day after end day 
+function getTaskWindowEdges(task) {
+    const a = task.triggers.map(t => {
+        t = task.triggerUnit === "weeks" ? t * 7 : t;
+        const start = t - task.pre;
+        const end = t + task.post;
+        return [start - 1, start, end, end + 1];
+    });
+
+    // return the flattened 1D array, with distinct members
+    return [...new Set([].concat.apply([], a))];
+}
+
+function range(a, b, d = 1) {
```

Comment:
whoa

Reply:
Changed the single letter variables, thanks.